src/reducers.js

```diff
@@ -100,6 +102,29 @@ function organisations(
   }
 }
 
+function usage (
+  state = {
+    raster_count: 0,
+    raster_total_size: 0,
+    scenario_count: 0,
+    scenario_total_size: 0,
+    timeseries_count: 0,
+    timeseries_total_size: 0,
+    isFetching: false,
+    timesFetched: 0,
```

Comment:
Is there a reason to use both snake cases and camel cases here? I guess because data comes from API is in snake case.

Reply:
Yes because it comes from the api. I choose this approach of changing as least as possible to the original data structure.